controllers/hardwareclassification_controller.go

```diff
@@ -18,6 +18,7 @@ package controllers
 import (
 	"context"
 	hwcc "hardware-classification-controller/api/v1alpha1"
+	utils "hardware-classification-controller/hcmanager"
```

Comment:
why renaming this to utils ? hcmanager or manager would be more descriptive probably

Reply:
@maelk Done